indy_node/server/config_req_handler.py

```diff
@@ -39,10 +35,37 @@ def doStaticValidation(self, request: Request):
             self._doStaticValidationPoolUpgrade(identifier, req_id, operation)
         elif operation[TXN_TYPE] == POOL_CONFIG:
             self._doStaticValidationPoolConfig(identifier, req_id, operation)
+        elif operation[TXN_TYPE] == AUTH_RULE:
+            self._doStaticValidationAuthRule(identifier, req_id, operation)
 
     def _doStaticValidationPoolConfig(self, identifier, reqId, operation):
         pass
 
+    def _doStaticValidationAuthRule(self, identifier, reqId, operation):
+        constraint = operation.get(CONSTRAINT)
+        ConstraintCreator.create_constraint(constraint)
+        action = operation.get(AUTH_ACTION, None)
+
+        if OLD_VALUE not in operation and action == EDIT_PREFIX:
+            raise InvalidClientRequest(identifier, reqId,
+                                       "Transaction for change authentication "
+                                       "rule for {}={} must contain field {}".
+                                       format(AUTH_ACTION, EDIT_PREFIX, OLD_VALUE))
+
+        if OLD_VALUE in operation and action == ADD_PREFIX:
+            raise InvalidClientRequest(identifier, reqId,
+                                       "Transaction for change authentication "
+                                       "rule for {}={} must not contain field {}".
+                                       format(AUTH_ACTION, ADD_PREFIX, OLD_VALUE))
+
+        auth_key = self.get_auth_key(operation)
+
+        if auth_key not in self.write_req_validator.auth_map and \
+                auth_key not in self.write_req_validator.anyone_can_write_map:
+            raise InvalidClientRequest(identifier, reqId,
+                                       "Key '{}' is not contained in the "
```

Comment:
I would make it more clear: `Unknown authorization rule: key '' is not found in authorization map`